airflow/utils/db.py

```diff
@@ -120,6 +120,9 @@ def initdb():
             conn_id='bigquery_default', conn_type='bigquery'))
     merge_conn(
         models.Connection(
+            conn_id='google_cloud_default', conn_type='google_cloud_platform'))
```

Comment:
Still needs to get cleaned up. Same declaration is here:

https://github.com/apache/incubator-airflow/pull/2390/files#diff-1fd56b45d5b414bd50b37baaac692c40R134